VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/individual-participant-status-list/individual-participant-status-list.component.html

```diff
@@ -99,7 +99,7 @@ <h2 class="govuk-heading-s">
             [innerHtml]="getHearingRole(participant)"
           ></dd>
 
-          <dd *ngIf="!isLoggedInParticipant(participant)" class="govuk-body-s govuk-!-margin-bottom-3">
```

Comment:
if we don't need this function anymore, then please delete it

Reply:
deleted